tests/roles/mariadb_copy/tasks/main.yaml

```diff
@@ -6,16 +6,78 @@
   ansible.builtin.include_tasks:
     file: env_vars_dst.yaml
 
+- name: start an adoption mariadb helper pod
+  ansible.builtin.shell: |-
+    {{ shell_header }}
+    {{ oc_header }}
+    {{ mariadb_copy_shell_vars_src }}
+
+    oc apply -f - <<EOF
+    ---
+    apiVersion: v1
+    kind: PersistentVolumeClaim
+    metadata:
+      name: mariadb-data
+    spec:
+      storageClassName: $STORAGE_CLASS
+      accessModes:
+        - ReadWriteOnce
+      resources:
+        requests:
+          storage: 10Gi
+    ---
+    apiVersion: v1
+    kind: Pod
+    metadata:
+      name: mariadb-copy-data
+      annotations:
+        openshift.io/scc: anyuid
```

Comment:
the same https://github.com/openstack-k8s-operators/data-plane-adoption/pull/316/files#r1511181345

Reply:
nvm, I agree with "I would suggest getting this done, and then do changes to clear out the network isolation thing separately?"